modules/python/src2/gen2.py

```diff
@@ -257,9 +258,11 @@ def __init__(self, name, decl=None, codegen=None):
         self.class_id = normalize_class_name(name)
 
         self.cname = name.replace(".", "::")
+        self.cname_of_value = self.cname
```

Comment:
Should be a property, because it can be evaluated on a fly and should not be changed.

```python
@property
def cname_of_value(self):
    return self.cname if self.issimple else "Ptr<" + self.cname + ">"
```

Reply:
Refactoring of existed coding style is out of scope for now.